scss/components/_responsive-embed.scss

```diff
@@ -29,7 +28,7 @@ $responsive-embed-ratio-widescreen: 16 by 9 !default;
 
 /// Creates a responsive embed container.
 /// @param {List} $ratio [$responsive-embed-ratio] - Ratio to use for the container, formatted as `x by y`.
-@mixin responsive-embed($ratio: $responsive-embed-ratio) {
+@mixin responsive-embed($ratio: 4 by 3) {
```

Comment:
Can you use `map-get($responsive-embed-ratios,default)` here (or a null variable you override after), instead of a magic number ?